src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataService.java

```diff
@@ -550,49 +552,26 @@ void deleteCase(UUID caseUUID, Boolean deleted) {
         log.info("Deleted Case: {} flag: {}", caseUUID, deleted, value(EVENT, CASE_DELETED));
     }
 
-    CaseSummary getCaseSummary(final UUID caseUUID) {
-        log.debug("Building CaseSummary for Case: {}", caseUUID);
-
-        final CaseData caseData = getCaseData(caseUUID);
-
-        CaseSummary.Builder summaryBuilder = new CaseSummary.Builder();
+    CaseSummary getCaseSummary(UUID caseUUID) {
+        var caseData = getCaseData(caseUUID);
 
-        summaryBuilder
-                .withCaseType(caseData.getType())
-                .withCreatedDate(caseData.getCreated().toLocalDate())
-                .withCaseDeadline(caseData.getCaseDeadline())
-                .withPrimaryCorrespondent(caseData.getPrimaryCorrespondent())
+        var summaryBuilder = new CaseSummary(caseData)
+                .withStageDeadlines(getStageDeadlines(caseData, caseData.getDataMap()))
                 .withPrimaryTopic(caseData.getPrimaryTopic())
-                .withActiveStages(caseData.getActiveStages());
-
-        Set<FieldDto> summaryFields = infoClient.getCaseSummaryFields(caseData.getType());
-
-        CaseActionDataResponseDto caseActionData = caseActionService.getAllCaseActionDataForCase(caseUUID);
-
-        summaryBuilder.withActions(caseActionData);
-
-        Map<String, String> caseDataMap = caseData.getDataMap();
-        summaryBuilder.withAdditionalFields(getAdditionalFieldsForSummary(summaryFields, caseDataMap));
-        summaryBuilder.withStageDeadlines(getStageDeadlines(caseData, caseDataMap));
-
-
-        auditClient.viewCaseSummaryAudit(caseData);
```

Comment:
This is an important decision to make - is this useful when to view the case summary you have already effectively viewed a case. The only reason this might be good is that you can load a case but not necessarily look at the summary tab.